pkgs/applications/system/howdy/default.nix

```
@@ -0,0 +1,135 @@
+{ stdenv
+, lib
+, bzip2
+, fetchFromGitHub
+, fetchurl
+, fmt
+, gettext
+, inih
+, installShellFiles
+, libevdev
+, meson
+, ninja
+, pam
+, pkg-config
+, python3
+}:
+
+let
+  data = let
+    baseurl = "https://github.com/davisking/dlib-models/raw/daf943f7819a3dda8aec4276754ef918dc26491f";
+  in {
+    "dlib_face_recognition_resnet_model_v1.dat" = fetchurl {
+      url = "${baseurl}/dlib_face_recognition_resnet_model_v1.dat.bz2";
+      sha256 = "0fjm265l1fz5zdzx5n5yphl0v0vfajyw50ffamc4cd74848gdcdb";
+    };
+    "mmod_human_face_detector.dat" = fetchurl {
+      url = "${baseurl}/mmod_human_face_detector.dat.bz2";
+      sha256 = "117wv582nsn585am2n9mg5q830qnn8skjr1yxgaiihcjy109x7nv";
+    };
+    "shape_predictor_5_face_landmarks.dat" = fetchurl {
+      url = "${baseurl}/shape_predictor_5_face_landmarks.dat.bz2";
+      sha256 = "0wm4bbwnja7ik7r28pv00qrl3i1h6811zkgnjfvzv7jwpyz7ny3f";
+    };
+  };
+
+  py = python3.withPackages (p: [
+    p.face_recognition
```

```suggestion
    p.face-recognition
```
this should fix eval